### DISCUSSION TITLE
No way to update to 2.7.0 in Home Assistant. Auto Updating not working

### DISCUSSION THREAD

**Discussion by superdong69** [mrlt8/docker-wyze-bridge#1083] [General]
As the title states.

Have this installed as a Home Assistant Add On. Correct repository configured, application watchdog and auto update enabled, does not update. There is no upgrade pathway in the addon UI, nor in the addon section for the bridge in Home Assistant.

As seen here:
https://i.imgur.com/PFUydvA.png

https://i.imgur.com/EGjRj0B.png

**Comment by superdong69** [comment]
Can anyone provide how to manually update this? Having major problems with my feeds reliability. It's piss poor. They all keep stopping, and not coming back online. I suspect an issue with 2.6.0 and the latest Home Assistant version.

**Reply by superdong69** [reply]
Possible to get an update on this @mrlt8 ?
I have tried uninstalling the addon, rebooting, and reinstalling from the addon store and I keep installing 2.6.0. When I click the check for update in UI, I get 2.7.0 available! message.

**Reply by mgrimace** [reply]
It seems to be a UI bug, and from your screenshot you seem to be on 2.7.0. This may also explain your other issues as there seems to be a lot of issues with the 2.7 release #1096 #1093 #1087 #1078 etc.

**Reply by superdong69** [reply]
While that may be true,  I don't seem to be having the issues posted,  my video streams just keep failing.
